### PR TITLE
Fixing Typing Error and some Punctuation Error

### PR DESCRIPTION
**Pull request by Rakesh9100**
## Fixed some punctuation errors and typos !!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ also able to generate compiled byte code in a C source file, see the "mrbtest"
 program under the "test" directory for an example.
 
 This achievement was sponsored by the Regional Innovation Creation R&D Programs
-of the Ministry of Economy, Trade and Industry of Japan.
+of the Ministry of Economy, Trade, and Industry of Japan.
```
Comment (matz) [inline README.md:19]: "the Ministry of Economy, Trade and Industry of Japan" is the official translation of the name of the [ministry](https://www.meti.go.jp/english/index.html). So we have to keep it.

File: README.md [hunk 2]
```diff
@@ -49,13 +49,13 @@ See the [compile.md](doc/guides/compile.md) file.
 
 There are two sets of documentation in mruby: the mruby API (generated by yard) and C API (Doxygen)
 
-To build both of them, simply go
+To build both of them, simply navigate:
```
Comment (matz) [inline README.md:52]: I am not an English native, but the word "navigate" makes me feel a bit awkward. Is it natural for command-line interaction?

File: README.md [hunk 3]
```diff
@@ -49,13 +49,13 @@ See the [compile.md](doc/guides/compile.md) file.
...
 
 ```
 rake doc
 ```
 
-You can also view them in your browser
+You can also view them in your browser by navigating:
```
Comment (matz) [inline README.md:58]: Same above regarding "navigate".

File: README.md (hunk 1 above)
Comment (Rakesh9100) [reply README.md:19]: I was bit confused and did it by mistake i think, Indeed you are right.

**Comment by fundamental** [comment]
I would say "To build both of them, simply run:". I wouldn't add the "by navigating:" to browser command as it's redundant. Also I don't know that "complying to STANDARD" should be altered to "complying with STANDARD" as I think the former is equally correct.

**Comment by Rakesh9100** [comment]
> I would say "To build both of them, simply run:". I wouldn't add the "by navigating:" to browser command as it's redundant. Also I don't know that "complying to STANDARD" should be altered to "complying with STANDARD" as I think the former is equally correct.

Yes, 'simply run' will also be good to say. You can change it to that. But talking about complying, it is good to have “complying with”. It is formally correct.

**Comment by matz** [comment]
According to the dictionary 'comply with' is the idiom. I am OK with the change. Or maybe we can use 'conform to' instead.

**Pull request closed without merging**